src/shipit_frontend/src/App/CodeCoverage.elm

```diff
@@ -164,28 +164,60 @@ viewDirectories directories =
             [ th [] [ text "Path" ]
             , th [] [ text "Previous" ]
             , th [] [ text "Current" ]
+            , th [] [ text "Difference" ]
             , th [] [ text "Bugs" ]
             ]
          ]
             ++ (List.map viewDirectory (Dict.toList directories))
         )
 
 
+roundFloat : Float -> Float
+roundFloat x =
+    toFloat (round (x * 100)) / 100
+
+
 viewDirectory : ( String, Directory ) -> Html Msg
 viewDirectory ( path, directory ) =
     let
+        diff =
+            case directory.previous of
+                Just previous ->
+                    roundFloat (directory.current - previous)
```

Comment:
Ah, so `previous` here is not the one defined on line 199, but the destructuring of directory.previous from line 184. Correct?